organization/_posts/2018-05-02-packaging.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "HSF Packaging Group Meeting #23, May 2, 2018"
+layout: default
+---
+
+# {{page.title}}
+
+Agenda
+======
+[https://indico.cern.ch/event/719557/](https://indico.cern.ch/event/719557/)
+
+Participants: Graeme Stewart, Pere Mato, Guilherme Amadio, Benedikt Hegner, Chris Burr, Ben Morgan, Liz Sexton-Kennedy, Raphal Pacholek, Emil Obreshkov, Ben Couturier, Geri Ganis, Chris Green, Giulio Eulisse, Patricia Mendez, Marco Clemencic, Lynn Garren, Jim Amundson, Patrick Gartung
+
+[Introduction - Graeme](https://indico.cern.ch/event/719557/contributions/2957742/attachments/1626928/2624333/HSF_Packaging_Group_Intro_2018-05-02.pdf)
+===========================
+- [Guix presentation tomorrow](https://indico.cern.ch/event/719851/)
+  - GNU reimplementation of Nix?
+    - Chris thinks it may be a fork from Nix in the past
+    - No non-GPL software allowed?
+  - Graeme will attend
+- [UK RSE](http://rse.ac.uk) - [2018 conference in Birmingham 3-4th September](http://rse.ac.uk/conf2018/)
+  - Abstracts submitted: one on HSF generally, and one on Packaging
+  - Will hear if accepted by end of May
+- Moving onto today’s agenda, as ever, contact Graeme and Ben if you have contributions, updates for future meetings
+
+[Support for multiple micro-architectures - Pere](https://indico.cern.ch/event/719557/contributions/2965980/attachments/1642767/2624258/HSF-Packaging-20180502.pdf)
+===============================================
+- Need separate stacks for instruction sets (e.g fma, avx etc)
+  - How to make available on CVMFS?
+  - Select stack at runtime from CPU detection
+  - Not all packages need/use these specialist instructions
+- Implementing this in LCGMake - recap given on recipes and views
+- Naming platforms - packages tagged on `arch-os-compiler-optimization`
+  - Extend arch with `arch+iset1+iset2+...+isetN` to describe which instruction sets
+    the package was compiled with
+- In LCGMake, new argument to package to select full instruction set build
+  (via compiler wrapper), which adds platform tag extension
+  - Some issues with static libraries and dependencies
```

Comment:
Maybe expand this to "issues with static libraries: rebuilding dependencies can pull in new instruction sets unwittingly, so this dependency needs to be expressed correctly in the dependent build"